charts/aws-fsx-csi-driver/templates/controller-deployment.yaml

```diff
@@ -48,10 +48,19 @@ spec:
           image: {{ printf "%s:%s" .Values.image.repository (default (printf "v%s" .Chart.AppVersion) (toString .Values.image.tag)) }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
+            {{- if ne .Release.Name "kustomize" }}
+            - --mode=controller
+            {{- else }}
+            # - --mode={all,controller,node} # specify the driver mode
```

Comment:
Should this line be commented out? As is, if the release name is not "kustomize" then we set the mode to controller, if it is "kustomize" then we doing nothing. What is the default behavior if mode isn't set? Should we even give the option for users to specify a driver mode of "node" in the controller?

Reply:
This was following precedent set by EBS, reviewing this with them to see if it should be removed

Reply:
Added a new commit that moves mode specification to values.yaml and removed this.